js/georgia-main.js

```diff
@@ -1070,6 +1070,24 @@ function onOptionsMenu(x, y) {
 	});
 	transportSizeMenu.appendTo(transportMenu);
 
+	const transportSpacingMenu = new Menu('Transport Button Spacing');
+	transportSpacingMenu.addRadioItems(['0px', '3px', '5px (default)', '8px', '15px', '20px', '+2'], pref.transport_buttons_spacing, [-1,3,5,8,15,20,999], (size) => {
+		if (size === -1) {
+			pref.transport_buttons_spacing -= 2;
+		} else if (size === 999) {
+			pref.transport_buttons_spacing += 2;
+		} else {
+			pref.transport_buttons_spacing = size;
+		}
+		createButtonImages();
+		createButtonObjects(ww, wh);
+		if (pref.show_transport_below) {

```

Comment:
We don't need this check as artwork size won't change based on spacing... just if the height of the buttons changes.